ts/src/test/test.ts

```diff
@@ -307,7 +309,12 @@ export default class testMainClass extends baseMainTestClass {
                     }
                 } else {
                     // if not a temporary connectivity issue, then mark test as failed (no need to re-try)
-                    dump ('[TEST_FAILURE]', exceptionMessage (e), exchange.id, methodName, argsStringified);
+                    if (isNotSupported) {
```

Comment:
was there a case when it was not supported? i mean, it should have automatically skip the method test if it is not `true` in `.has`.

Reply:
@ttodua yeah the code was reaching this point that's why I added this